components/multiselect/multiselect.jsx

```diff
@@ -238,8 +242,8 @@ export default class MultiSelect extends React.Component {
                             onChange={this.onChange}
                             value={this.props.values}
                             valueRenderer={this.props.valueRenderer}
-                            menuRenderer={() => null}
-                            arrowRenderer={() => null}
+                            menuRenderer={this.handleRender}
```

Comment:
No need for a function here, actually you could even remove the props

Reply:
I've tried to remove those but
- when `menuRenderer` is removed, the menu opened up as soon as the user access a component with MultiSelect (see screenshot)
![screen shot 2017-12-19 at 1 16 44 am](https://user-images.githubusercontent.com/5334504/34122552-ccbe2a74-e467-11e7-90d8-24e2766aea23.png)
- when `arrowRenderer`, an extra caret show up in the text box 
![screen shot 2017-12-19 at 2 20 48 am](https://user-images.githubusercontent.com/5334504/34122610-0bf4dbf2-e468-11e7-9b6a-64726b0e64af.png)

So I'll keep it that way.